src/test/test_nvmeof_mon_encoding.cc

```diff
@@ -101,22 +110,24 @@ void test_MNVMeofGwMap() {
   dout(0) << "False pending map: " << pending_map << dendl;
 
   auto msg = make_message<MNVMeofGwMap>(pending_map);
-  msg->encode_payload(0);
+  dout(0) << "before encode msg: " << *msg << dendl;
+  msg->encode_payload(CEPH_FEATURES_ALL);
+  dout(0) << "after encode msg: " << *msg << dendl;
   msg->decode_payload();
-  dout(0) << "decode msg: " << *msg << dendl;
+  dout(0) << "after decode msg: " << *msg << dendl;
 
-  dout(0)   << "\n == Test GW Delete ==" << dendl;
-  pending_map.cfg_delete_gw("GW1" ,group_key);
-  dout(0) << "deleted GW1 " << pending_map << dendl;
+  //dout(0)   << "\n == Test GW Delete ==" << dendl;
```

Comment:
assuming these would be removed before merging?

Reply:
No, @leonidc wants them commented out as described in the commit message.